azurefunctions-extensions-bindings-blob/azurefunctions/extensions/bindings/blob/utils.py

```
@@ -2,11 +2,27 @@
 #  Licensed under the MIT License.
 import os
 
+from azure.identity import DefaultAzureCredential, ManagedIdentityCredential
+from azure.storage.blob import BlobServiceClient
+
+
+def validate_connection_setting(connection_string: str) -> str:
+    """
+    Validates and returns the connection setting. The setting must
+    not be None - if it is, a ValueError will be raised.
+    """
+    if connection_string is None:
+        raise ValueError(
+            "Storage account connection string cannot be None. "
+            "Please provide a connection string."
+        )
+    else:
+        return connection_string
+
```

Moved this logic out of `get_connection_string` so that we can fail faster if the customer does something like `connection=None` or forgets to include the `connection` property. This is validating the connection setting, not the connection string.